src/pages/storage/forms/StoragePoolFormMain.tsx

```diff
@@ -203,6 +207,68 @@ const StoragePoolFormMain: FC<Props> = ({ formik }) => {
               />
             </>
           )}
+          {isPureDriver && (
+            <>
+              <Input
+                {...formik.getFieldProps("pure_api_token")}
+                type="text"
+                label="API Token"
+                placeholder="Enter Pure Storage API token"
+                help="API token with admin access to the Pure Storage array."
+                onChange={(e) => {
+                  ensureEditMode(formik);
+                  formik.handleChange(e);
+                }}
+                required
+              />
+              <Input
+                {...formik.getFieldProps("pure_gateway")}
+                type="text"
+                label="API Gateway"
+                placeholder="Enter Pure Storage API gateway"
+                help="The Pure Storage URL where API is accessible."
+                onChange={(e) => {
+                  ensureEditMode(formik);
+                  formik.handleChange(e);
+                }}
+                required
+              />
+              <Select
+                {...formik.getFieldProps("pure_gateway_verify")}
+                label="Verify Gateway"
+                help="Whether to verify the PureStorage certificate. False for self-signed certs"
```

Comment:
```suggestion
                help="Whether to verify the PureStorage certificate. Set this option to false for self-signed certs that are not in the LXD trust store."
```
Not 100% sure if the addition is correct. I think there is a way to add the custom cert to LXD, maybe check with Din if this wording is correct.

Reply:
Verifying!